pkg/printers/table/collector.go

```diff
@@ -211,6 +216,9 @@ func (r *ResourceStateCollector) processApplyEvent(e event.ApplyEvent) {
 		klog.V(4).Infof("%s apply event not found in ResourceInfos; no processing", identifier)
 		return
 	}
+	if e.Error != nil {
+		previous.Error = e.Error
```

Comment:
does this mean we only show the first error?
are there cases of multiple errors after validation?

Reply:
We currently only attempt once, so we should only get one event pertaining to apply/prune of a specific resource. If we were to attempt more than once, I think it would make sense to use the error from the latest attempt.

Reply:
eh. ok. looks weird, but it's not a big deal.